docs/multiple-tests/jsdoc_check-examples/patterns.xml

```diff
@@ -0,0 +1,6 @@
+<module name="root">
+    <module name="jsdoc_check-examples" />
+    <module name="BeforeExecutionExclusionFileFilter">
+        <property name="fileNamePattern" value="\.eslintrc\.js" />
```

Comment:
shouldn't you be forcing the "checkEslintConfig" or whatever is called? on this test?

Reply:
No, because that we already know it doesn't work.
We're testing that the default is correct to not make the tool fail. If then the users want to force this behaviour, it's their responsability to provide a supported .eslintrc file